packages/next-frontend/src/components/step/index.tsx

```diff
@@ -18,9 +18,14 @@ export function Step({
     disabled,
     children,
 }: StepProps) {
+    const wrapperRef = useRef<HTMLDivElement>(null);
     const [open, setOpen] = useState(false);
-    const [height, setHeight] = useState<Height>(0);
-    const [heightAnimationDone, setHeightAnimationDone] = useState(true);
+    const [spring, animate] = useSpring(
+        () => ({
+            height: "83px",
```

Comment:
Could we maybe use REMs here?

Reply:
Using rems here causes strange behavior on the height.